fapolicy_analyzer/ui/types.py

```diff
@@ -0,0 +1,25 @@
+# Copyright Concurrent Technologies Corporation 2023
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+from enum import Enum
+
+
+class PAGE_SELECTION(Enum):
+    TRUST_DATABASE_ADMIN = "trust"
+    RULES_ADMIN = "rules"
+    ANALYZE_FROM_AUDIT = "audit log"
```

Comment:
This should be "debug log".

We have had some ambiguity between debug/audit in the past.  It doesnt need fixed now other than what is user facing here.

We updated the other user facing part in #771

Reply:
@dorschs57 I did a /audit/debug/ for this